services/state/blockrecords/block_info.proto

```diff
@@ -0,0 +1,67 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2018 - 2023 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import "timestamp.proto";
+
+option java_package = "com.hederahashgraph.api.proto.java";
+// <<<pbj.java_package = "com.hedera.hapi.node.state.blockrecords">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+
+/**
+ * Information about ongoing, most recently completed, and last 256 blocks.
+ */
+message BlockInfo {
+  /**
+   * The last block number, this is the last completed immutable block.
+   */
+  int64 last_block_number = 1;
+  /**
+   * The consensus time of the first transaction of the last block, this is the last completed immutable block.
+   */
+  Timestamp first_cons_time_of_last_block = 2;
```

Comment:
To my knowledge, I haven't talked to anyone that can explain why this property is necessary, or if it's even intentional. I thought it was a mistaken duplication of `first_cons_time_of_current_block`.